FreeRTOS/iot-reference-esp32c3
# Update for 202407.00 release information

Description
-----------
* Update manifest for release information
* Update CHANGELOG information
* Update release.yml file

Test Steps
-----------
Test release in forked repository - https://github.com/chinglee-iot/iot-reference-esp32c3/releases/tag/v202407.00

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
